src/admin/permission-groups/views/PermissionGroupDetail.tsx

```diff
@@ -0,0 +1,194 @@
+
+import { redirectToClientPage } from '../../../authentication/helpers/redirects';
+import { PERMISSION_GROUP_PATH, PERMISSIONS_TABLE_COLS } from '../permission-group.const';
+import { GET_PERMISSION_GROUP_BY_ID } from '../permission-group.gql';
+import { PermissionGroupService } from '../permission-group.service';
+import { Permission, PermissionGroup, PermissionsTableCols } from '../permission-group.types';
+
+interface PermissionGroupEditProps extends DefaultSecureRouteProps<{ id: string }> {}
+
+const PermissionGroupEdit: FunctionComponent<PermissionGroupEditProps> = ({ history, match }) => {
+	const [t] = useTranslation();
+
+	// Hooks
+	const [permissionGroup, setPermissionGroup] = useState<PermissionGroup | null>(null);
+	const [loadingInfo, setLoadingInfo] = useState<LoadingInfo>({ state: 'loading' });
+	const [sortColumn, sortOrder, handleSortClick] = useTableSort<PermissionsTableCols>('label');
+
+	const initOrFetchPermissionGroup = useCallback(async () => {
+		try {
+			const response = await dataService.query({
+				query: GET_PERMISSION_GROUP_BY_ID,
+				variables: { id: match.params.id },
+			});
+
+			const permissionGroupObj = get(response, 'data.users_permission_groups[0]');
+
+			if (!permissionGroupObj) {
+				setLoadingInfo({
+					state: 'error',
+					icon: 'search',
+					message: t('Deze permissie groep werd niet gevonden'),
+				});
+				return;
+			}
+
+			const permissions: Permission[] = flatten(
+				get(permissionGroupObj, 'permission_group_user_permissions', []).map(
+					(permissionGroupItem: any) => {
+						return get(permissionGroupItem, 'permissions', []);
+					}
+				)
+			);
+
+			const permGroup = {
+				permissions,
+				id: permissionGroupObj.id,
+				label: permissionGroupObj.label,
+				description: permissionGroupObj.description,
+				created_at: permissionGroupObj.created_at,
+				updated_at: permissionGroupObj.updated_at,
+			};
+			setPermissionGroup(permGroup);
+		} catch (err) {
+			console.error(
+				new CustomError('Failed to get permission group by id', err, {
+					query: 'GET_PERMISSION_GROUP_BY_ID',
+					variables: { id: match.params.id },
+				})
+			);
+			setLoadingInfo({
+				state: 'error',
+				message: t('Het ophalen van de permissie groep is mislukt'),
+			});
+		}
+	}, [setLoadingInfo, setPermissionGroup, t, match.params.id]);
+
+	useEffect(() => {
+		initOrFetchPermissionGroup();
+	}, [initOrFetchPermissionGroup]);
+
+	useEffect(() => {
+		if (permissionGroup) {
+			setLoadingInfo({ state: 'loaded' });
+		}
+	}, [permissionGroup, setLoadingInfo]);
+
+	const handleEditClick = () => {
+		redirectToClientPage(
+			buildLink(PERMISSION_GROUP_PATH.PERMISSION_GROUP_EDIT, {
+				id: match.params.id,
+			}),
+			history
+		);
+	};
+
+	const renderTableCell = (rowData: Permission, columnId: PermissionsTableCols) => {
+		switch (columnId) {
+			case 'label':
+			case 'description':
+				return rowData[columnId];
+
+			case 'actions':
```

Comment:
Kan deze niet weggelaten worden of komt hier in de toekomst nog iets te staan?
Anders best nog een TODO zetten.